LOAN/LoanContracts/LoanHMDA.rdf

```diff
@@ -166,6 +155,13 @@ Copyright (c) 2015-2018 Object Management Group, Inc.</sm:copyright>
 		<fibo-fnd-utl-av:adaptedFrom>the Revised HMDA regulatory text.</fibo-fnd-utl-av:adaptedFrom>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-loan-ln-hmda;HowSubmitted">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-cls;Classifier"/>
+		<rdfs:label>how submitted</rdfs:label>
+		<skos:definition>category indicating whether the applicant or borrower submitted the application for the covered loan directly to the reporting financial institution</skos:definition>
```

Comment:
if it's really that specific the name should be more specific too

Reply:
Agreed, and may go away.  I moved it because the individuals were in this ontology, but didn't look at the content at all